src/invocable.rs

```diff
@@ -345,6 +347,20 @@ macro_rules! method {
 /// ```
 #[macro_export]
 macro_rules! call {
+    ($cls_name:literal :: $fn_name:literal ($( $args:expr ),*) -> $rett:ty) => {
+        (|| {
+            $crate::RttiSystem::get()
+                .get_class($crate::types::CName::new($cls_name))
+                .ok_or($crate::InvokeError::ClassNotFound($cls_name))?
+                .static_methods()
+                .iter()
+                .find(|x| x.as_function().name() == $crate::types::CName::new($fn_name)
+                || x.as_function().short_name() == $crate::types::CName::new($fn_name))
```

Comment:
```suggestion
                .find(|x| x.as_function().name() == $crate::types::CName::new($fn_name))
```
I think the short name comparison shouldn't be done here, because of 2 reasons:
- natives are going to have the same name and short name
- it would be inconsistent with `call!` for instance methods

Reply:
switched for helper method in 21bcaa9.